main/ertp/guide/contract-hosts.md

```diff
@@ -11,3 +13,60 @@ Some seats provide the methods `getWinnings()` and `getRefunds()`, which return
 promises for payments for the outputs of the interaction. We provide the
 `collect()` method (described later) to simplify collection of winnings and
 refunds into appropriate purses.
+
+## Installations
+
+An installation of a Contract can spawn multiple copies each with the same or different terms. Each spawned instance has distinct invites and distinct seats representing a specific group of agents interacting according to the same prescribed roles.
+
+### <span style="color:red">Possible typo on the next sentence?</span>
```

Comment:
@katelynsills Do you know what the sentence below is trying to say? I think there's a typo.

"The Installation can also have functions with names starting 'check', as defined by the contract."

Reply:
The installation can also provide helper functions, useful for validating taht the parameters to the contract match the expectations of the person trying to make use of it. These functions must have names starting 'check' so examples might be checkSeat() or checkAuctionInstance(). The contract gets to define them, and the parameters are specific to the particular contract.

Reply:
Thanks for clarifying. I updated the wording:

The Installation can also have functions with names starting with 'check'. These are helper functions that are specific to the particular contract (E.g. Escrow has checkUnits() and checkPartialUnits()).